vendor/assets/javascripts/meltdown/jquery.meltdown.js

```diff
@@ -485,6 +485,9 @@
             this.bar =  $('<div class="meltdown_bar"></div>').appendTo(this.wrap);
             this.editorWrap =  $('<div class="' + plgName + '_editor-wrap" />').appendTo(this.wrap);
             this.editorDeco =  $('<div class="' + plgName + '_editor-deco" />').appendTo(this.editorWrap);
+
```

Comment:
Since we're modifying a third-party library here, can you make a big block comment at the top of this file warning future users that this is a modified version. Also make a comment above this line indicating that it is our modification and should be copied over manually to any future versions that we upgrade to.